### ISSUE TITLE
rss shouldnt modify the urls in the tweets

### ISSUE THREAD

**Issue by sajiljosephs** [zedeus/nitter#811]
rss shouldnt modify the urls in the tweets
Do we have an option not to modify the tweets and urls's while fetching it from rss
Eg: <description><![CDATA[<p>This week on <a href="[https://nitter.net/search?q=%23SpaceToGround">#SpaceToGround</a>](https://nitter.net/search?q=%23SpaceToGround%22%3E#SpaceToGround%3C/a%3E), Crew-5 splashes down, and a cargo delivery arrives with two new heart-related <a href="[https://nitter.net/ISS_Research"](https://nitter.net/ISS_Research%22) title="ISS Research">@ISS_Research</a> investigations that could help future space explorers and patients on Earth! 💓🛰️</p><img src="[https://nitter.net/pic/media%2FFrbk5APXgAIh7Fi.jpg"](https://nitter.net/pic/media%2FFrbk5APXgAIh7Fi.jpg%22) style="max-width:250px;" />]]></description>

All the URL's are modified to nitter.net

https://github.com/zedeus/nitter/blob/93f605f4fe9f0787e3dd618cba85130b9cba2438/src/utils.nim#L13

I am asking, so that I can modify them in my local nitter instance

**Comment by zedeus** [comment]
They're modified because that's how nitter works. You can disable it by changing the default replacements to empty strings:
https://github.com/zedeus/nitter/blob/bc219aa792cc0e4117888b2036a969559f4f2689/nitter.example.conf#L39-L41

**Comment by sajiljosephs** [comment]
Made the changes as you suggested. Everything looks good other than images url. (<img src="https://twitter.com/pic/card_img%2F1636824194678661121%2FQ-KdPFuo%3Fformat%3Djpg%26name%3D800x419" style="max-width:250px;" />)


While checking nitter.net also the url seems to be different.
Original URL in twitter: https://pbs.twimg.com/card_img/1636824194678661121/Q-KdPFuo?format=jpg&name=small >> Works
Nitter generated URL: https://nitter.net/pic/card_img%2F1636824194678661121%2FQ-KdPFuo%3Fformat%3Djpg%26name%3D800x419 >> Works
Custom nitter conf replacing nitter to twitter:  https://twitter.com/pic/card_img%2F1636824194678661121%2FQ-KdPFuo%3Fformat%3Djpg%26name%3D800x419 >> Doesnt Work

pbs.twimg.com shouldnt be replaced with anything. Do we have a conf entry for it ?

**Comment by zedeus** [comment]
No, nitter isn't intended to be used this way. You'll have to look for where image urls are created, maybe using encodeUrl.

**Comment by sajiljosephs** [comment]
Thanks, Can you please let me know the format it is writing while converting to rss. I am asking about the format or software with version like atom or json or rss

**Comment by sajiljosephs** [comment]
After the suggested changes the image src is still incorrect.

<img src="https://twitter.comhttps://pbs.twimg.com/card_img/1638307456076988424/mgiAsf4t?format=jpg&name=800x419"

**Comment by sajiljosephs** [comment]
Not only images, Can you suggest the changes not to modify the entire tweet content?

**Comment by zedeus** [comment]
I did, just combine it. If you want an easier solution use rsshub or something.

**Comment by sajiljosephs** [comment]
Sorry, I didnt understand. Inorder to correct the img src, what modification needs to be done ?
<img src="[https://twitter.comhttps://pbs.twimg.com/card_img/1638307456076988424/mgiAsf4t?format=jpg&name=800x419](https://twitter.comhttps//pbs.twimg.com/card_img/1638307456076988424/mgiAsf4t?format=jpg&name=800x419)"

**Comment by zedeus** [comment]
https://github.com/zedeus/nitter/blob/bc219aa792cc0e4117888b2036a969559f4f2689/src/views/rss.nimf#L42

There's a bunch of these in this file that generates RSS. You need to remove the `${urlPrefix}` part
